delfin/api/schemas/storage_capabilities_schema.py

```diff
@@ -76,7 +76,7 @@
                                                 'maxLength': 255}
                             },
                         },
-                        'writeRequests': {
+                        'writeIops': {
```

Comment:
What about writeIOPS/readIOPS
why enum? for unit

Reply:
As discussed We follow the camelCase for all metrics..